File: python/problem9.py
import math
def pythagorean_triplet():
  a = 2
  while (a < 500):
    b = 2
    while (b < 500):
      c = math.sqrt(a ** 2 + b ** 2)
      if (a + b + c == 1000 and c % 1 == 0):
        return a*b*c
      b += 1
    a += 1


print pythagorean_triplet()

File: ruby/problem9.rb
def pythagorean_formula(a,b)
  Math.hypot(a,b)

end

def triplet
  i = 2
  while i < 500
    j = 2
    while j < 500
      hypotenuse = pythagorean_formula(i,j)

      if hypotenuse% 1 == 0
       sum = i +j + hypotenuse
       return i*j*hypotenuse if sum == 1000
      end
      j += 1
    end
   i += 1
  end
  
end

p triplet
